packages/authgear-react-native/src/ui_implementation.ts

```diff
@@ -1,19 +1,23 @@
 import { openAuthorizeURL, openAuthorizeURLWithWebView } from "./nativemodule";
 
 /**
+ * OpenAuthorizationURLOptions is options for {@link UIImplementation.openAuthorizationURL}.
+ *
  * @public
  */
 export interface OpenAuthorizationURLOptions {
-  /*
-   * The URL to open.
+  /**
+   * The target authorization URL.
    */
   url: string;
-  /*
-   * The URL to detect.
+  /**
+   * The value should be a valid Redirect URI to which the response will be sent after authentication.
+   * You must also add a Redirect URI in Authgear Poral via the Redirect URI section of your Authgear Application.
    */
```

Comment:
Change the description to `The URL to be detected by the UIImplementation. When this URL is detected, the UIImplementation MUST return this URL, and close itself`.

Reply:
ok